### PR TITLE
Ability to Create Sub Assets via RPC. Fix large Asset Transactions

### PR DESCRIPTION
**Pull request by blondfrogs**

### CODE REVIEW

File: src/rpc/assets.cpp [hunk 1]
```diff
@@ -179,6 +179,132 @@ UniValue issue(const JSONRPCRequest& request)
     return result;
 }
 
+UniValue issuesubasset(const JSONRPCRequest& request)
+{
+    if (request.fHelp || !AreAssetsDeployed() || request.params.size() < 2 || request.params.size() > 8)
```
Comment (cfox) [inline src/rpc/assets.cpp]: Shouldn't this be `<3`?

File: src/rpc/assets.cpp [hunk 2]
```diff
@@ -179,6 +179,132 @@ UniValue issue(const JSONRPCRequest& request)
     return result;
 }
 
+UniValue issuesubasset(const JSONRPCRequest& request)
+{
+    if (request.fHelp || !AreAssetsDeployed() || request.params.size() < 2 || request.params.size() > 9)
```
Comment (cfox) [inline src/rpc/assets.cpp]: should be `< 3` ?

File: src/script/standard.cpp [hunk 3]
```diff
@@ -72,40 +72,49 @@ bool Solver(const CScript& scriptPubKey, txnouttype& typeRet, std::vector<std::v
...
-        return true;
-    }
-
-    if (scriptPubKey.IsReservedAsset()) {
-        typeRet = TX_RESERVED_ASSET;
-        std::vector<unsigned char> hashBytes(scriptPubKey.begin()+3, scriptPubKey.begin()+23);
-        vSolutionsRet.push_back(hashBytes);
-        return true;
-    }
+//    if (scriptPubKey.IsNewAsset()) {
```
Comment (cfox) [inline src/script/standard.cpp]: kill comments

Comment (blondfrogs) [reply src/script/standard.cpp]: Done

File: src/rpc/assets.cpp (hunk 1 above)
Comment (blondfrogs) [reply src/rpc/assets.cpp]: yep.

**Pull request merged**